packages/commons/src/email-manager/emailManager.ts

```diff
@@ -54,6 +66,40 @@ export function initPecEmailManager(
         html: body,
       });
     },
+    sendWithAttachments: async (
```

Comment:
This function looks to be equal to `send` but with one more parameter, could it make sense to just add the parameter  as optional to `send`?

Reply:
I know. The "problem" is that here I could have used a single function but we have a common interface between `PecEmailSender` and `SesMailManager` and if for the Pec we use the same method to send an email with attachments. With the SesMailManager I have to implement a workaround so I prefer to create a dedicated function.